api_volontaria/apps/user/tests/tests_view_users.py

```diff
@@ -89,6 +85,54 @@ def test_profile(self):
             permissions
         )
 
+    def test_profile_admin(self):
+        self.client.force_authenticate(user=self.admin)
+        response = self.client.get(
+            'http://api.example.org/users/' + str(self.user.id)
+        )
+
+        # HTTP code is good
+        self.assertEqual(
+            response.status_code,
+            status.HTTP_200_OK,
+            response.content
+        )
+
+        # Number of results is good
+        content = json.loads(response.content)
+
+        self.check_attributes(content, self.ATTRIBUTES + ['private_note', ])
```

Comment:
It's a good usage but in order to keep it clear for future developer i propose to create a new `ATTRIBUTES_ADMIN` variable on the top of the file like this:

```
self.ATTRIBUTES_ADMIN = self.ATTRIBUTES + ['private_note', ]
```

It will allow you to do 

```
self.check_attributes(content, self.ATTRIBUTES_ADMIN)
```

And should be more easy to rampup for future developer and avoid duplication in the future.